### ISSUE TITLE
Import date is not available - 705

### ISSUE THREAD

**Issue by Romeo-PHILLIPS** [osm-search/Nominatim#2864]
**Describe the bug**

When querying `/status.php?format=json` I get a response:

```
{
    "status": 705,
    "message": "Import date is not available"
}
```
The first issue is that the documentation doesn't talk about this status code (https://nominatim.org/release-docs/latest/api/Status/)
<img width="590" alt="Capture d’écran 2022-11-04 à 17 36 05" src="https://user-images.githubusercontent.com/92714362/200028336-bf8b3440-f8cd-4cc2-a6c2-5838db0309cc.png">

The second issue, is how to fix that status code? Note that all queries work fine.

**Comment by Romeo-PHILLIPS** [comment]
I enabled replication and it fix the 705

**Comment by Romeo-PHILLIPS** [comment]
https://github.com/osm-search/Nominatim/issues/705

**Comment by lonvia** [comment]
You get this error when you had no internet connection during the import. It's not fatal, the rest of the database will still work fine. It just won't tell you about the state of the data.

**Comment by lonvia** [comment]
Documentation fixed in #2865.
